### PR TITLE
[FB-551] Fix BOT_OWNER permission for Bots which belong to a Team

### LINKED ISSUE #551: Discord Teams break BOT_OWNER Permission
if a bot is transferred to a Discord Team no one will have the BOT_OWNER permission anymore. 

Tested by transferring my test bot to Fred's Team and neither of us has BOT_OWNER permission over it

### PR DESCRIPTION
**Pull request by Nanabell**
Fixes #551 until discordapp/discord-api-docs#787 allows querying for team members

### CODE REVIEW

File: FredBoat/src/main/java/fredboat/perms/PermsUtil.kt [hunk 1]
```diff
@@ -40,6 +40,10 @@ object PermsUtil {
     suspend fun getPerms(member: Member): PermissionLevel = when {
         member.sentinel.applicationInfo.ownerId == member.id
         -> PermissionLevel.BOT_OWNER // https://fred.moe/Q-EB.png
+
+        //TODO: This is a kinda "fix" for Bots that belong to a Team since we currently can't query the members in a team as a bot (https://github.com/discordapp/discord-api-docs/issues/787)
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/perms/PermsUtil.kt]: This is not meant to be temperary, and should not be `//TODO`

File: FredBoat/src/main/java/fredboat/perms/PermsUtil.kt [hunk 2]
```diff
@@ -40,6 +40,10 @@ object PermsUtil {
     suspend fun getPerms(member: Member): PermissionLevel = when {
         member.sentinel.applicationInfo.ownerId == member.id
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/perms/PermsUtil.kt]: Should be moved to `isBotOwner`

File: FredBoat/src/main/java/fredboat/perms/PermsUtil.kt [hunk 3]
```diff
@@ -89,6 +93,13 @@ object PermsUtil {
         }
     }
 
+    private fun isBotOwner(member: Member): Boolean {
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/perms/PermsUtil.kt]: This method could be reduced to a one-liner. Why not `ownerIds.contains(member.id)`?

File: config/templates/fredboat.yml [hunk 4]
```diff
@@ -54,6 +54,7 @@ config:
                                   # things happening to your bot in the selfhosting chat you will be publicly taunted.
   prefix:            '<<'         # Default prefix used by the bot
   botAdmins:         []           # Add comma separated userIds and roleIds that should have access to bot admin commands. Find role ids with the ;;roleinfo command
+  botOwners:         []           # Add comma separated userIds that should have access to bot OWNER commands. Mainly intended for Bots that belong to teams
```
Comment (Frederikam) [inline config/templates/fredboat.yml]: "Bots" should be lowercase. You and your silly German grammar :eyes:

Comment (Nanabell) [reply config/templates/fredboat.yml]: LOL

File: FredBoat/src/main/java/fredboat/perms/PermsUtil.kt [hunk 5]
```diff
@@ -38,8 +38,6 @@ import javax.annotation.CheckReturnValue
 object PermsUtil {
 
     suspend fun getPerms(member: Member): PermissionLevel = when {
-        member.sentinel.applicationInfo.ownerId == member.id
-        -> PermissionLevel.BOT_OWNER // https://fred.moe/Q-EB.png
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/perms/PermsUtil.kt]: Hey, don't remove the memes :v

File: FredBoat/src/main/java/fredboat/perms/PermsUtil.kt [hunk 6]
```diff
@@ -91,12 +89,8 @@ object PermsUtil {
         }
     }
 
-    private fun isBotOwner(member: Member): Boolean {
-        for (id in Launcher.botController.appConfig.ownerIds) {
-            if (member.id == id) return true
-        }
-        return false
-    }
+    private fun isBotOwner(member: Member): Boolean = Launcher.botController.appConfig.ownerIds.contains(member.id) or
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/perms/PermsUtil.kt]: `or` should be `||`

**Pull request merged**